src/aitsi/m3spin/pkb/exception/SimplePkbException.java

```diff
@@ -0,0 +1,7 @@
+package aitsi.m3spin.pkb.exception;
+
+public class SimplePkbException extends Exception{
+    public SimplePkbException(String err) {
+        super("Error while creating AST tree: " + err);
```

Comment:
Jeśli w treści wyjątku jest "Error while creating AST", to wyjątek powinien się nazywać np. `AstCreationException`. Nazwa `PkbException` (już po usunięciu tego Simple) wskazuje że jest to generyczny wyjątek dla PKB - więc nie powinien odnosić się konkretnie do AST, bo podejrzewam, że będzie wyrzucany nie tylko w AST.
Najlepszym rozwiązaniem wydaje się być rozdzielenie tego na 2 wyjątki

- `PkbException` z generycznym komunikatem, albo nawet bez
- `AstCreationException` rozszerzający `PkbException` z komunikatem `"Error while creating AST: " + err`